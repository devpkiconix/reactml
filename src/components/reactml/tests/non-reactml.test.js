import React from 'react';

import renderer from 'react-test-renderer';

test('Test non-reactml renders correctly', () => {
    const tree = renderer
        .create(<span>
            <span>hello world</span>
        </span>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
