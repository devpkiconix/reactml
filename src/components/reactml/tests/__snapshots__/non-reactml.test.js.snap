// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test non-reactml renders correctly 1`] = `
<span>
  <span>
    hello world
  </span>
</span>
`;
